src/sources/aws_s3/sqs.rs

```diff
@@ -253,13 +254,18 @@ impl IngestorProcess {
         let messages = self.receive_messages().await;
         let messages = messages
             .map(|messages| {
+                let byte_size = messages.iter().map(std::mem::size_of_val).sum();
+                emit!(&BytesReceived {
```

Comment:
Do we need to emit `BytesReceived` for bytes that do not actually end up as an event? From my (potentially mistaken) understanding, these SQS messages are only used to figure out which S3 objects to request and do not end up as part of events themselves.

Reply:
Yeah, I was gonna comment the same thing. I think the metrics here should all be oriented around events that actually end up flowing through the topology. That is, we should not count bytes and messages from SQS, just what we read from AWS S3 objects.

Reply:
good point, I will drop that 😉